ts/sagas/backendInfo.ts

```diff
@@ -20,9 +20,12 @@ const BACKEND_INFO_LOAD_INTERVAL = 60 * 60 * 1000;
 const BACKEND_INFO_RETRY_INTERVAL = 60 * 60 * 10 * 1000;
 
 function* backendInfoWatcher(): IterableIterator<Effect> {
+  console.warn(apiUrlPrefix);
   const backendPublicClient = BackendPublicClient(apiUrlPrefix);
 
-  function getServerInfo(): Promise<BasicResponseType<ServerInfo> | undefined> {
+  function getServerInfo(): Promise<
+    t.Validation<BasicResponseType<ServerInfo> | undefined>
```

Comment:
the new client will never return `undefined` (`undefined` gets replaced by `Left<Errors, T>`)